apps/laboratory/tests/email.spec.ts

```diff
@@ -1,63 +1,94 @@
-import { expect } from '@playwright/test'
-import { testMEmail } from './shared/fixtures/w3m-email-fixture'
+import { expect, test, type BrowserContext } from '@playwright/test'
+import { ModalWalletPage } from './shared/pages/ModalWalletPage'
+import { Email } from './shared/utils/email'
+import { ModalWalletValidator } from './shared/validators/ModalWalletValidator'
 import { SECURE_WEBSITE_URL } from './shared/constants'
-import type { ModalWalletPage } from './shared/pages/ModalWalletPage'
-import type { ModalWalletValidator } from './shared/validators/ModalWalletValidator'
 
-testMEmail.beforeEach(async ({ modalValidator }) => {
-  await modalValidator.expectConnected()
+/* eslint-disable init-declarations */
+let page: ModalWalletPage
+let validator: ModalWalletValidator
+let context: BrowserContext
+/* eslint-enable init-declarations */
+
+// -- Setup --------------------------------------------------------------------
+const emailTest = test.extend<{ library: string }>({
+  library: ['wagmi', { option: true }]
+})
+
+emailTest.describe.configure({ mode: 'serial' })
```

Comment:
what if we wrap the tests with a [`describe`](https://playwright.dev/docs/api/class-test#test-describe)?